app/views/static_pages/index.html.erb

```diff
@@ -45,7 +45,7 @@
         <%= link_to "Статуси замовлень",'#'%>
       </li>
       <li>
-        <%= link_to "Користувачі",'#'%>
```

Comment:
@IgorKrv please revert it 
![image](https://user-images.githubusercontent.com/6030146/50322511-ece19100-04dd-11e9-8a21-46a82189fb53.png)

Reply:
@IgorKrv why you didn't write feedback to me?

Reply:
@IgorKrv ???